typescript/sdk/src/ism/types.ts

```diff
@@ -42,6 +42,7 @@ export enum IsmType {
   OP_STACK = 'opStackIsm',
   ROUTING = 'domainRoutingIsm',
   FALLBACK_ROUTING = 'defaultFallbackRoutingIsm',
+  ICA_FALLBACK_ROUTING = 'icaFallbackRoutingIsm',
```

Comment:
I think the "fallback" qualifier implies there is another version
I would prefer `ICA` or `ICA_ROUTING`